src/Components/TagPickerGridComponent/TagPickerGridComponent/css/TagPickerGridComponent.css

```diff
@@ -1,8 +1,18 @@
-.tagPickerGridComponent {
+.tagPickerGridBodyContainer {
```

Comment:
if simple enough, ms-tagitem could be slightly increased to match the lookup field padding (2px vs. 4px).